File: Scripts/Listener/IShareListener.cs

// *******************************************
// Company Name:	深圳市晴天互娱科技有限公司
//
// File Name:		IShareListener.cs
//
// Author Name:		Bridge
//
// Create Time:		2024/04/28 14:43:29
// *******************************************

namespace Instagram.Api
{
	/// <summary>
	/// 
	/// </summary>
	public interface IShareListener
	{
		void OnSuccess();

		void OnError(int errCode, string errMsg);
	}
}

File: Plugins/Android/com/bridge/ins/IShareListener.java
package com.bridge.ins;

/**
 * 分享监听
 */
public interface IShareListener {
    /**
     * 分享成功
     */
    void onSuccess();

    /**
     * 分享失败
     * @param errCode 错误码
     * @param errMsg 错误信息
     */
    void onError(int errCode, String errMsg);
}
